node/web/templatetags/node_extras.py

```diff
@@ -0,0 +1,10 @@
+from django import template
+
+from node.core.utils.cryptography import derive_public_key, get_signing_key
+
+register = template.Library()
+
+
+@register.simple_tag
+def get_node_identifier():
+    return derive_public_key(get_signing_key())
```

Comment:
call `node.core.utils.cryptography.get_node_identifier`

Reply:
I removed "node" word from function name to avoid  same name as get_node_identifier.
```
from core.utils.cryptography import get_node_identifier

@register.simple_tag
def get_identifier():
    return get_node_identifier()
```